src/erlmld_batch_processor.erl

```diff
@@ -80,7 +80,7 @@ initialize(Opts, ShardId, ISN) ->
         maps:merge(Defaults, Opts),
     State =
         #state{flusher_mod = FlusherMod,
-               flusher_state = FlusherMod:init(ShardId, FlusherModData),
+               flusher_state = erlmld_flusher:init(FlusherMod, ShardId, FlusherModData),
```

Comment:
Just curious: what's the improvement in this change?

Reply:
Two off the top of my head:
- It allows `dialyzer` and `xref` to check the functions.
- It allows developers to find the calls to the callbacks easily (since they're *only* in the module that defines the callbacks).

Reply:
cool, thx 👍